kubernetes/update_images.py

```diff
@@ -60,6 +65,15 @@ def main():
             print(f"Using existing local copy of {image_name}")
             continue
 
+        prereqs = IMAGE_PREREQS.get(image_type, [])
+        for prereq in prereqs:
```

Comment:
Alternative would be to form something like `to_build = prereqs + [build_request]` and then iterate over `to_build`. This would reduce code duplication but would mean we'd have less informative print messages -- not sure which is better, will let you decide (or perhaps there's a best of both worlds)

Reply:
The `tag` is also different — we want to build `:cpp`, not `:<git_hash>-cpp` in order for `:cpp-and-twogtp` to be able to import the C++ image correctly.
I guess we could iterate over a list where each element is a tuple `(<message to print>, <tag>, <image type>)`? I'll leave the code as is for now

Reply:
Oh, right, I missed that. Yeah, let's leave it as is then.